README.md

```diff
@@ -36,15 +36,34 @@ For entitlements, only users from those groups are selectable which have to be c
 
 ![advanced permission entitlement](screenshots/aclAdmin.png)
 
+## Notes
+
+* Currently using encryption on group folders is not supported. All files stored within a group folder will be stored unencrypted.
+* In Client applications, group folders will appear as external storage and may need to be explicitly addressed for download.
+
+## Command line configuration via occ
+
+Group folders can be configured on the command line (cli) using the `occ` command:
+
+`occ groupfolders:create <name>` &rarr; create a group folder
+`groupfolders:delete <folder_id> [-f|--force]` &rarr; delete a group folder and all its contents
+`occ groupfolders:expire` &rarr; trigger file version expiration (see [Nextcloud docs](https://docs.nextcloud.com/server/latest/admin_manual/configuration_files/file_versioning.html) for details)
```

Comment:
I don't fully get the necessity for this command. Are expirations for group folders not handled by the normal background job but have to be triggered manually?
Shall we recommend configuring a dedicated cron job to auto-expire group folders' objects?

Reply:
There is a background job so I would agree that his is not really needed. I guess @icewind1991 knows more about that :wink: